File: src/main/java/com/rtomyj/skc/config/DateConfig.java
package com.rtomyj.skc.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Lazy;

import java.text.SimpleDateFormat;

@Configuration
@Lazy
public class DateConfig
{
	@Bean("dbSimpleDateFormat")
	public static SimpleDateFormat getDBSimpleDateFormat()
	{
		return new SimpleDateFormat("yyyy-MM-dd");
	}
}

File: src/main/java/com/rtomyj/skc/config/DateConfig.kt
package com.rtomyj.skc.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Lazy
import java.text.SimpleDateFormat

@Configuration
@Lazy
class DateConfig {
	@Bean("dbSimpleDateFormat")
	fun dBSimpleDateFormat(): SimpleDateFormat = SimpleDateFormat("yyyy-MM-dd")
}